tasks/php.js

```diff
@@ -8,8 +8,9 @@ var objectAssign = require('object-assign');
 
 module.exports = function (grunt) {
 	var checkServerTries = 0;
+	var serverStarted = false;
 
-	function checkServer(hostname, port, path, cb) {
+	function checkServer(hostname, port, path, done) {
```

Comment:
To make it clearer what this does (= signal to grunt that the task is done), I had trouble understanding waht `cb` meant.
I used the same name as grunt docs: http://gruntjs.com/api/inside-tasks